torchvision/datasets/utils.py

```diff
@@ -202,70 +208,41 @@ def download_file_from_google_drive(file_id: str, root: str, filename: Optional[
     """
     # Based on https://stackoverflow.com/questions/38511444/python-download-files-from-google-drive-using-url
```

Comment:
Maybe we can drop this comment at some point, because our implementation no longer resembles the answer in the link.